### ISSUE TITLE
Missing values for "Daily Battery Charging Energy"

### ISSUE THREAD

**Issue by NeoGen85** [alangibson/homeassistant-sungrow#25]
In my setup with a SH10RT, I can see the values for "Daliy Battery Charinging Energy" in the Inverter WebUI, however the sensor for that value is missing from HA.

The only sensor I can find is "Daily Battery Charing Energy from PV", which does not let me see how much power from the grid I've put in to my battery.

I've looked through the code, and there is no register for "Daliy Battery Charinging Energy". Is this by design..?

![HA_missing_sensor](https://github.com/user-attachments/assets/5966dcde-3467-4930-b5f1-f339e35cc43c)
![inverter_ui](https://github.com/user-attachments/assets/82b8f912-8bdd-489e-8ac3-02b2cc182fe7)

**Comment by NeoGen85** [comment]
I found that in @AlexanderLanin repo, the register actually exists. However, since this repo was updated last 5 days ago, and his 7 months ago, I figured you'd want to add this to your code?


This is from his repo:

```
- name: "daily_battery_charge_energy"
    group: has_battery
    level: 1
    address: 13040
    data_type: U16
    accuracy: 0.1
    unit_of_measurement: "kWh"
```

**Comment by alangibson** [comment]
Its already there under a slightly different name:

    - name: "daily_charge_energy"    
      level: 2
      address: 13040
      datatype: "U16"
      accuracy: 0.1
      unit: "kWh"

which ends up being this sensor:

    SungrowInverterSensorEntityDescription(
        key=SUNGROW_DAILY_BATTERY_CHARGE_GRID_ENERGY ,
        register=SUNGROW_DAILY_BATTERY_CHARGE_GRID_ENERGY ,
        name="Daily Battery Charge Energy",
        native_unit_of_measurement=UnitOfEnergy.KILO_WATT_HOUR,
        device_class=SensorDeviceClass.ENERGY,
        state_class=SensorStateClass.TOTAL_INCREASING,
    ),

But then there is also:

    - name: "daily_battery_charge_from_pv"
      level: 1
      address: 13012
      datatype: "U16"
      accuracy: 0.1
      unit: "kWh"

which ends up being:

    SungrowInverterSensorEntityDescription(
        key=SUNGROW_DAILY_BATTERY_CHARGE_PV_ENERGY,
        register=SUNGROW_DAILY_BATTERY_CHARGE_PV_ENERGY,
        name="Daily Battery Charge PV Energy",
        native_unit_of_measurement=UnitOfEnergy.KILO_WATT_HOUR,
        device_class=SensorDeviceClass.ENERGY,
        state_class=SensorStateClass.TOTAL_INCREASING,
    ),

So basically there are two different registers that can end up being read depending on your inverter model. I don't have a battery so I can't debug much past that.
